app/helpers/verification_helper.rb

```diff
@@ -50,19 +50,29 @@ def display_verification_type_name(v_type)
     end
   end
 
-  # method added specifically to handle the displaying of 'Alive Status'
+  # method searches for the verification type change history in the verification type, either in the type_history_elements or history_tracks
+  # if the verification type has ever been in 'outstanding' status, whether historically or currently, it will return true
+  def had_outstanding_status?(verif_type)
+    previous_states = fetch_previous_states(verif_type)
+    previous_states_history = fetch_previous_states_history(verif_type, previous_states)
+    outstanding_status?(previous_states, previous_states_history, verif_type)
+  end
+
+  # method added specifically to handle the displaying of 'Alive Status' and 'American Indian Status'
   # this verification type should always be visible to admin
-  # but should only display for consumers/brokers/broker agency staff if the validation_status is 'outstanding'
+  # but should only display for consumers/brokers/broker agency staff if the validation_status has ever been in 'outstanding'
+  # or if the feature for the verification type is disabled
   def can_display_type?(verif_type)
-    return true unless verif_type.type_name == 'Alive Status'
-    return false unless EnrollRegistry.feature_enabled?(:enable_alive_status)
-    return true if current_user.has_hbx_staff_role?
-
-    previous_states = verif_type.type_history_elements.pluck(:from_validation_status).compact
-    return true if previous_states.include?('outstanding')
-    return true if verif_type.validation_status == 'outstanding'
+    return true if current_user.has_hbx_staff_role? || had_outstanding_status?(verif_type)
```

Comment:
I think we need to move the `had_outstanding_status?` check into the `case` statement below.  Thinking about the following scenario:
- user is a consumer
- verification type never had outstanding status
- `ai_an_self_attestation` is disabled

Then the AI/AN verification type would not display.

I think we can change line 72 to account for this:
```
!EnrollRegistry.feature_enabled?(:ai_an_self_attestation) || had_outstanding_status?(verif_type)
```

And similar, but slightly different for Alive Status:
```
EnrollRegistry.feature_enabled?(:alive_status) && had_outstanding_status?(verif_type)
```